gazu/task.py

```diff
@@ -81,6 +81,15 @@ def all_tasks_for_asset(asset):
     return sort_by_name(tasks)
 
 
+@cache
+def all_tasks_for_episode(episode):
+    """
+    Retrieve all tasks directly linked to given episode.
+    """
+    tasks = client.fetch_all("episodes/%s/tasks" % episode["id"])
```

Comment:
Can you use the normalize helper for the episode parameter? It allows to give either id or dict as a parameter for the episode parameter.

Reply:
Done